libraries/AP_HAL_ChibiOS/hwdef/HitecMosaic/hwdef.dat

```diff
@@ -172,5 +172,5 @@ define HAL_PERIPH_ENABLE_BARO
 define HAL_GPS_TYPE_DEFAULT 10
 define HAL_GPS_COM_PORT_DEFAULT 3
 define GPS_SBF_STREAM_NUMBER 3
-define GPS_SBF_EXTRA_CONFIG "sdfa, DSK1, DeleteOldest\n", "sfn, DSK1, Incremental, Mosaic\n", "sfno, off\n"
+define GPS_SBF_EXTRA_CONFIG "sdfa, DSK1, DeleteOldest", "sfn, DSK1, Incremental, Mosaic\n", "sfno, off"
```

Comment:
shouldn't the \n after Mosaic be removed too ?

Reply:
Yeah, I missed it with the multiple items to a line here. It doesn't actually change the build size due to alignment reasons, but it's still better to remove it here. Fixed it up.